_search-plugins/neural-sparse-search.md

```diff
@@ -16,8 +16,8 @@ Introduced 2.11
 
 When selecting a model, choose one of the following options:
 
-- Use a sparse encoding model at both ingestion time and search time (high performance, relatively high latency).
-- Use a sparse encoding model at ingestion time and a tokenizer at search time for relatively low performance and low latency. The tokenism doesn't conduct model inference, so you can deploy and invoke a tokenizer using the ML Commons Model API for a more consistent experience.
+- Use a sparse encoding model at both ingestion time and search time for high search relevance with relatively high latency.
+- Use a sparse encoding model at ingestion time and a tokenizer at search time for low search latency with relatively low search relevance. The tokenism doesn't conduct model inference, so you can deploy and invoke a tokenizer using the ML Commons Model API for a more consistent experience.
```

Comment:
```suggestion
- Use a sparse encoding model at ingestion time and a tokenizer at search time for lower search latency at the expense of relatively worse search relevance. Tokenization doesn't involve model inference, so you can deploy and invoke a tokenizer using the ML Commons Model API for a more consistent experience.
```

Reply:
Changed "worse" to "lower" with regard to "relevance".